ui/templates/provision.j2

```diff
@@ -160,6 +160,14 @@
                               title="Joyent Minimal Native Zone">
                               joyent-minimal
                             </option>
+                            {% if builder_brand %}
```

Comment:
Wow!  Good!  Curious to know how the testing works out here.

Reply:
Latest version works!